### DISCUSSION TITLE
Tutorial Video

### DISCUSSION THREAD

**Discussion by Randomguyawesome** [edward-ly/stepmania-song-manager#7] [Ideas]
I would love to see a tutorial video on how to use this I haven't had much luck using it but I would love to use this program I see it came with packs already which are downloaded and being used currently but would love to add more if you could make a tutorial video that would be appreciated a lot :D

**Comment by edward-ly** [comment]
A tutorial on how to download packs or how to add packs not already in the app? Either way, good idea, but may not happen for a while depending on how much free time I have. For now, let me know if you have any questions about the [README](https://github.com/edward-ly/stepmania-song-manager#readme) documentation.
